### PR TITLE
kcov : init at 29

### PR DESCRIPTION
**Pull request by FlorentBecker**

### CODE REVIEW

File: pkgs/development/tools/analysis/kcov/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  src = fetchurl {
+    url = "https://github.com/SimonKagstrom/kcov/archive/v${version}.tar.gz";
+    sha256 = "0nspf1bfq8zv7zmcmvkbgg3c90k10qcd56gyg8ln5z64nadvha9d";
+  };
+
+  buildInputs = [ cmake pkgconfig libelf zlib curl elfutils python libiberty binutils ];
+  
+  meta = with stdenv.lib; {
+    description = "code coverage tester for compiled programs, Python scripts and shell scripts.";
```
Comment (pSub) [inline pkgs/development/tools/analysis/kcov/default.nix]: `description` should not end with a period.

File: pkgs/development/tools/analysis/kcov/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    longDescription =
+      ''  Kcov is a code coverage tester for compiled programs, Python
```
Comment (pSub) [inline pkgs/development/tools/analysis/kcov/default.nix]: Nitpick: Could you get rid of the extra spaces?

**Comment by pSub** [comment]
Looks good otherwise.

**Comment by FlorentBecker** [comment]
fixed

**Pull request merged**